### PR TITLE
fix: make rollup to proplery generate bundled.js and demo/*.min.js

### PR DESCRIPTION
**Pull request by sun-mota**
# Alaska Airlines Pull Request

## Before Submitting this pull request:
- Link all tickets in this repository related to this PR in the `Development` section
_note: all pull requests require at least one linked ticket_
- If this PR is `Ready For Review`, all ticket's linked under `Development` must have their status changed to `Ready For Review` as well

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license and I have performed a self-review of my own update.**

## Summary by Sourcery

Bug Fixes:
- Fix the Rollup configuration to properly generate bundled.js and demo/*.min.js files.

### CODE REVIEW

File: packages/config/src/rollup.config.mjs [hunk 1]
```diff
@@ -40,29 +29,18 @@ const createConfig = (name, input, output) => ({
   ]
 });
 
-const mainConfig = createConfig('formkit', './index.js', 'dist');
-const componentConfigs = getComponentEntryPoints().map(({ name, input, output }) =>
-  createConfig(name, input, output)
-);
+const mainConfig = createConfig('formkit', './src/index.js', 'dist');
```
Comment (chrisfalaska) [inline packages/config/src/rollup.config.mjs:32]: Do we need this anymore? I don't think this file exists in the monorepo?

**Review by chrisfalaska — CHANGES_REQUESTED** [review]

File: packages/config/src/rollup.config.mjs (hunk 1 above)
Comment (sun-mota) [reply packages/config/src/rollup.config.mjs:32]: this config runs under each component, `src/index.js` still exists for each components

File: packages/config/src/rollup.config.mjs [hunk 2]
```diff
@@ -40,29 +29,18 @@ const createConfig = (name, input, output) => ({
...
 
-
-function createExampleConfig(componentName, entryPoint) {
+function createExampleConfig(entryPoint) {
   return {
     input: {
-      [`${entryPoint}.min`]: `./components/${componentName}/demo/${entryPoint}.js`,
+      [`${entryPoint}.min`]: `./demo/${entryPoint}.js`,
```
[the remark this answers is not in the document]
Comment (sun-mota) [reply packages/config/src/rollup.config.mjs:37]: this rollup config runs under each component's folder. path from the root does not work.

File: packages/config/src/rollup.config.mjs [hunk 3]
```diff
@@ -1,68 +1,15 @@
...
 }
-
-const docConfig = getComponentEntryPoints().map( ({ name }) => {
-  const indexExample = createExampleConfig(name, 'index');
-  const apiExample = createExampleConfig(name, 'api');
-  return [indexExample, apiExample];
-}).flat();
-
-export default [mainConfig, ...componentConfigs, ...docConfig];
+export default [mainConfig, createExampleConfig('index'), createExampleConfig('api')];
```
Comment (chrisfalaska) [inline packages/config/src/rollup.config.mjs]: I get an error:

```
Cannot find name 'mainConfig'.ts(2304)
```

If we no longer need it, we can just remove it:

`export default [createExampleConfig('index'), createExampleConfig('api')];`

**Review by chrisfalaska — CHANGES_REQUESTED** [review]

File: packages/config/src/rollup.config.mjs (hunk 3 above)
Comment (sun-mota) [reply packages/config/src/rollup.config.mjs:44]: fixed!

**Review by chrisfalaska — APPROVED** [review]
Looks good! 🚀

**Pull request merged**